### PR TITLE
doc: add examples to readme

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,255 @@
...
+[![Twitter](https://img.shields.io/twitter/url.svg?label=Follow%20%40ariga%2Fatlas&style=social&url=https%3A%2F%2Ftwitter.com%2Fariga_io)](https://twitter.com/ariga_io)
+[![Discord](https://img.shields.io/discord/930720389120794674?label=discord&logo=discord&style=flat-square&logoColor=white)](https://discord.com/invite/zZ6sWVg6NT)
+
+<img width="30%" align="right" style="display: block; margin:40px auto;"
+src="https://atlasgo.io/uploads/images/gopher.png"/>
+
+Atlas is a tool for managing and migrating database schemas using modern DevOps principles. It offers two workflows:
+
+- **Declarative**: Similar to Terraform, Atlas compares the current state of the database with the desired state defined in
+an HCL or SQL schema, and generates a migration plan to reach that state.
```
Comment (hilakashai) [inline README.md:12]: link HCL ?

**Review by hilakashai — COMMENTED** [review]
Beast 🚀

File: README.md [hunk 2]
```diff
@@ -1,52 +1,257 @@
...
+
+<img width="30%" align="right" style="display: block; margin:40px auto;"
+src="https://atlasgo.io/uploads/images/gopher.png"/>
+
+Atlas is a tool for managing and migrating database schemas using modern DevOps principles. It offers two workflows:
+
+- **Declarative**: Similar to Terraform, Atlas compares the current state of the database with the desired state defined in
+an [HCL] or SQL schema, and generates a migration plan to reach that state.
+- **Versioned**: Unlike other tools, Atlas intelligently plans schema migrations for you. Users can describe their desired
+database schema in HCL or SQL and use Atlas CLI to plan, lint, and apply the necessary migrations.
```
Comment (rotemtam) [inline README.md:14]:
```suggestion
- **Versioned**: Unlike other tools, Atlas automatically plans schema migrations for you. Users can describe their desired
database schema in HCL or SQL and use Atlas CLI to plan, lint, and apply the necessary migrations.
```

File: README.md [hunk 3]
```diff
@@ -1,52 +1,257 @@
...
+Click [here](https://atlasgo.io/getting-started#installation) to read instructions for other platforms.
 
 ## Getting Started
-Get started with Atlas by following the [Getting Started](https://atlasgo.io/cli/getting-started/setting-up) docs.
+Get started with Atlas by following the [Getting Started](https://atlasgo.io/getting-started/) docs.
 This tutorial teaches you how to inspect a database, generate a migration plan and apply the migration to your database.
 
-## Features
-- **Inspecting a database**: easily inspect your database schema by providing a database URL.
+## Key Features:
```
Comment (rotemtam) [inline README.md:42]:
```suggestion
## Key features:
```

File: README.md [hunk 4]
```diff
@@ -1,52 +1,257 @@
...
+  ]
+}
+```
+</details>
+
+Inspect a specific PostgreSQL schema and get its representation in SQL DDL syntax:
+```shell
+atlas schema inspect \
+  --url "postgres://root:pass@:5432/test?search_path=public&sslmode=disable" \
+  --log '{{ sql . }}'
```
Comment (rotemtam) [inline README.md:108]: 😍

File: README.md [hunk 5]
```diff
@@ -1,52 +1,257 @@
...
+
+```sql
+-- create "users" table
+CREATE TABLE "users" ("id" integer NULL, ...);
+-- create "posts" table
+CREATE TABLE "posts" ("id" integer NULL, ...);
+```
+</details>
+
+## Schema Diff
```
Comment (rotemtam) [inline README.md:121]:
```suggestion
## `schema diff`
```

File: README.md [hunk 6]
```diff
@@ -1,52 +1,257 @@
...
+
+Compare an HCL schema with an SQL schema:
+```shell
+atlas schema diff \
+  --from file://schema.sql \
+  --to file://schema.hcl \ 
+  --dev-url docker://postgres/15/test  
+````
+
+## Schema Apply
```
Comment (rotemtam) [inline README.md:164]:
```suggestion
## `schema apply`
```

Comment (rotemtam) [reply README.md:164]: or `## Applying schemas`

File: README.md (hunk 5 above)
Comment (rotemtam) [reply README.md:121]: Or `## Calculating schema diffs`

File: README.md [hunk 7]
```diff
@@ -1,52 +1,257 @@
...
+  --url mysql://root:pass@:3306/db1 \
+  --to "file://migrations?version=20221118091226" \
+  --dev-url docker://mysql/8/db1
+```
+
+### Additional `schema` commands
+Atlas offers additional commands to assist users managing their database schemas. These include `schema clean` and
+`schema fmt`. For more information, see the versioned migration documentation at https://atlasgo.io/declarative/inspect.
+
+## Migrate Diff
```
Comment (rotemtam) [inline README.md:201]:
```suggestion
## `migrate diff`
```
Or
`## Automatic migration planning`

**Review by rotemtam — APPROVED** [review]
Great job
see some suggestions

**Pull request merged**